patternlab/styleguide/source/_patterns/01-atoms/09-media/video.twig

```diff
@@ -14,6 +14,6 @@
         {% include "@atoms/decorative-link.twig" %}
       </div>
     {% endif %}
-    <iframe width="{{ video.width }}" height="{{ video.height }}" src="{{ video.src }}" frameborder="0" allowfullscreen aria-label="{{ video.label }}"></iframe>
+    <iframe width="{{ video.width }}" height="{{ video.height }}" src="{{ video.src }}" frameborder="0" allowfullscreen allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" aria-label="{{ video.label }}"></iframe>
```

Comment:
I'd like to ensure video wouldn't play automatically. Can you change  `autoplay;` to `autoplay: 'none';` in the `allow` attribute value?

Reply:
removed autoplay from the iframe per your comment

Reply:
Thank you.